insights/specs/sos_archive.py

```diff
@@ -77,6 +77,7 @@ class SosSpecs(Specs):
     docker_list_images = simple_file("sos_commands/docker/docker_images")
     docker_storage = simple_file("/etc/sysconfig/docker-storage")
     dumpe2fs_h = glob_file("sos_commands/filesys/dumpe2fs_-h_*")
+    eap_json_reports = glob_file(r"/var/tmp/insights-runtimes/uploads/*.json")
```

Comment:
I doubt if the files collected by sos are files also modified in last `24` hours?